README.md

```diff
@@ -1,9 +1,11 @@
 # lob_elixir (alpha)
 
 [![Build Status](https://travis-ci.org/lob/lob-elixir.svg?branch=master)](https://travis-ci.org/lob/lob-elixir)
+[![Hex.pm](https://img.shields.io/hexpm/v/lob_elixir.svg)](https://hex.pm/packages/lob_elixir)
 [![Coverage Status](https://coveralls.io/repos/github/lob/lob-elixir/badge.svg?branch=master)](https://coveralls.io/github/lob/lob-elixir?branch=master)
+[![Libraries.io for releases](https://img.shields.io/librariesio/release/hex/lob_elixir.svg)](https://libraries.io/hex/lob_elixir)
 
-Elixir Library for [Lob API](https://lob.com/)
+Elixir library for [Lob API](https://lob.com/).
 
 ### This is an ALPHA library for the Lob API.
```

Comment:
Do we still want this `alpha library` information here?

Reply:
Hmm, good question. My understanding was that the library was in "alpha" until docs were published on lob.com and we released it as `1.0.0`. I guess we can consider this a "beta"?